### PR TITLE
Admin UI: Setup enmeshed design

### PR DESCRIPTION
**Pull request by stamenione**
# Readiness checklist

-   [ ] I added/updated unit tests.
-   [ ] I added/updated integration tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

### CODE REVIEW

File: AdminUi/apps/admin_ui/lib/main.dart [hunk 1]
```diff
@@ -68,13 +78,38 @@ class MainApp extends StatelessWidget {
...
   @override
   Widget build(BuildContext context) {
-    const colorSchemeSeed = Color.fromARGB(255, 23, 66, 141);
-    return MaterialApp.router(
-      debugShowCheckedModeBanner: false,
-      themeMode: ThemeMode.light,
-      theme: ThemeData(colorSchemeSeed: colorSchemeSeed, useMaterial3: true),
-      darkTheme: ThemeData(brightness: Brightness.dark, colorSchemeSeed: colorSchemeSeed, useMaterial3: true),
-      routerConfig: _router,
+    return DynamicColorBuilder(
```
Comment (jkoenig134) [inline AdminUi/apps/admin_ui/lib/main.dart:81]: as we do not want to harmonize our colors in code you can remove the dynamic color builder and just add the dark scheme from `color_schemes.dart`

File: AdminUi/apps/admin_ui/lib/theme/colors/custom_color.dart
Comment (jkoenig134) [inline AdminUi/apps/admin_ui/lib/theme/colors/custom_color.dart:1]: this file should be called `custom_colors.dart`

File: AdminUi/apps/admin_ui/lib/main.dart [hunk 2]
```diff
@@ -68,13 +78,38 @@ class MainApp extends StatelessWidget {
...
+
+          darkScheme = darkDynamic.harmonized();
+          darkCustomColors = darkCustomColors.harmonized(darkScheme);
+        } else {
+          lightScheme = lightColorScheme;
+          darkScheme = darkColorScheme;
+        }
+
+        return MaterialApp.router(
+          themeMode: Provider.of<ThemeManager>(context).themeMode,
```
Comment (jkoenig134) [inline AdminUi/apps/admin_ui/lib/main.dart:98]: `themeMode: ThemeMode.system`

File: AdminUi/apps/admin_ui/pubspec.yaml [hunk 3]
```diff
@@ -9,11 +9,13 @@ environment:
 dependencies:
   admin_api_sdk: ^1.0.0
   admin_api_types: ^1.0.0
+  dynamic_color: ^1.7.0
   flutter:
     sdk: flutter
   flutter_svg: ^2.0.10+1
   get_it: ^7.6.7
   go_router: ^13.2.0
+  provider: ^6.1.2
```
Comment (jkoenig134) [inline AdminUi/apps/admin_ui/pubspec.yaml:18]: please remove these added libs

**Review by jkoenig134 — CHANGES_REQUESTED** [review]

File: AdminUi/apps/admin_ui/pubspec.yaml (hunk 3 above)
Comment (stamenione) [reply AdminUi/apps/admin_ui/pubspec.yaml]: Removed

File: AdminUi/apps/admin_ui/lib/main.dart (hunk 2 above)
Comment (stamenione) [reply AdminUi/apps/admin_ui/lib/main.dart]: ThemeMode.system is a default value for router.

**Review by jkoenig134 — APPROVED** [review]

File: AdminUi/apps/admin_ui/lib/main.dart [hunk 4]
```diff
@@ -63,17 +67,23 @@ final _router = GoRouter(
   ],
 );
 
-class MainApp extends StatelessWidget {
-  const MainApp({super.key});
+class AdminUIApp extends StatelessWidget {
```
Comment (jkoenig134) [inline AdminUi/apps/admin_ui/lib/main.dart:70]: I think the casing in this project is AdminUi

Comment (stamenione) [reply AdminUi/apps/admin_ui/lib/main.dart:70]: Ah sorry didn't read it correctly.

**Review by jkoenig134 — APPROVED** [review]

**Review by tnotheis — APPROVED** [review]

**Pull request merged**